discovery-infra/test_infra/utils/libvirt_utils.py

```diff
@@ -0,0 +1,182 @@
+import json
+import logging
+
+import libvirt
+import xml.dom.minidom as md
+
+import waiting
+
+from logger import log
+from test_infra import consts
+
+# TODO - Temporary imports - will be removed after deleting all libvirt methods from utils.py 
+import filelock
+import os
+from contextlib import contextmanager
+
+
+# TODO - Temporary copied from utils to prevent cyclic import - 
+#  will be removed after deleting all libvirt methods from utils.py
+@contextmanager
+def _file_lock_context(filepath='/tmp/discovery-infra.lock', timeout=300):
```

Comment:
it is not part of libvirt, move it to another utils

Reply:
Read the TODO comment above